### ISSUE TITLE
waybackmachine

### ISSUE THREAD

**Issue by treaki** [diegocr/CleanLinks#74]

**Comment by treaki** [comment]
ups sry, clicked wrong button...

**Comment by diegocr** [comment]
You can already whitelist any domain you'd like by right-clicking the toolbar button.

**Comment by treaki** [comment]
yes i know that i am able to do settings by my own... But http://web.archive.bibalex.org/web/ is the mirror of the central waybackmachine http://web.archive.org/web/ and i cant remember that i had that problem there. so its looked for me like that is white listed for everyone in the global data file suppled by the addon. And so its mirror should be dealed with the same way. even if i'm mistaken in that assumption...:

Never the less that is a miss behavior in data processing in general, i requested one archived file and get forwarded to the original site, that is caused in my opinion by that addon and should that way handled also by the developers of that eddon...

thanks in advance for the fix and keep up the good work
